src/sass/components/_tour.scss

```diff
@@ -37,24 +37,73 @@
 
 .popover-wrapper {
   position: absolute;
-  top: -10px;
+  top: -($spacing-small);
+
+  .epic-progress & {
+    right: -($spacing-small);
+    top: -($spacing-medium);
+  }
 
   .logo-popover & {
     top: unset;
   }
 
+  .metecho-breadcrumb & {
+    top: -($var-spacing-x-small);
+  }
+
+  .next-steps & {
```

Comment:
@jgerigmeyer This app is being translated into other languages, right?  If so, then we won't be able to position these popovers based on where the current English text ends up, like this one. 

@dvdherron I think for block level elements like headings, we probably want to position the popover target before the content.

Reply:
Yes, I think that's correct. We shouldn't do any positioning strictly based on the length of text (in current font and language).

Reply:
Makes sense. For consistency's sake then we should probably have all of the popovers placed on the same general side of the elements they are referencing, right? @stacyk 

I'm working on removing them from inside of heading elements based on your comment below.